pkg/bootstrap/docker/openshift/ansible.go

```diff
@@ -0,0 +1,182 @@
+package openshift
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"text/template"
+
+	"github.com/golang/glog"
+
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	"github.com/openshift/origin/pkg/bootstrap/docker/run"
+)
+
+const (
+	defaultAnsibleImageUser      = "root"
+	defaultOpenshiftAnsibleImage = "openshift-ansible"
```

Comment:
In order to use the pattern we establish with the other images (e.g. origin-logging-elasticsearch) we will need @sdodson  team to build an image with a similiar name (e.g. origin-openshift-ansible).  This is the only way someone may pass in image-prefix and have it apply to all the images.  The alternative is we need to add a separate swtich for ansible image and prefix.  I believe if you follow the code it will utilized the same defaults the other images use.  I am open to suggestions.

Regarding one on docker, I can push one there if you want to test.  I had to build my own based on changes required to the openshift-ansible codebase